src/platform_impl/windows/event_loop.rs

```diff
@@ -1792,18 +1847,18 @@ unsafe fn public_window_callback_inner<T: 'static>(
                 window_state.scale_factor = new_scale_factor;
 
                 if new_scale_factor == old_scale_factor {
-                    return 0;
+                    return LRESULT(0);
                 }
 
                 window_state.fullscreen.is_none()
                     && !window_state.window_flags().contains(WindowFlags::MAXIMIZED)
             };
 
-            let style = winuser::GetWindowLongW(window, winuser::GWL_STYLE) as _;
-            let style_ex = winuser::GetWindowLongW(window, winuser::GWL_EXSTYLE) as _;
+            let style = WINDOW_STYLE(GetWindowLongW(window, GWL_STYLE) as u32);
+            let style_ex = WINDOW_EX_STYLE(GetWindowLongW(window, GWL_EXSTYLE) as u32);
```

Comment:
There's `util::adjust_window_rect()` for these already, which queries the same just before calling into `util::adjust_window_rect_with_styles()`.

Reply:
`style` and `style_ex` get used again, further down.